Sources/UIPanGestureRecognizer.swift

```diff
@@ -96,6 +96,10 @@ open class UIPanGestureRecognizer: UIGestureRecognizer {
         if state == .changed {
             onAction?()
         }
+
+        if cancelsTouchesInView {
+            trackedTouch.hasBeenCancelledByAGestureRecognizer = true
+        }
```

Comment:
I do think it makes sense to do this here.

Is it relevant to the logic to have this at the end of the function, or is this something that could (also?) happen earlier? I wonder whether we might have timing bugs doing it at the end of the function like this?

Reply:
I think it doesn't matter because `trackedTouch.hasBeenCancelledByAGestureRecognizer` gets only used for the UIView lifecycles which always gets executed after running the actions on the recognizers

Reply:
Ok awesome. It would be useful if you could add that as a comment (maybe that assumption will be broken in a later code change and it could help debugging)